<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Billetterie</class>
 <widget class="QDialog" name="Billetterie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Repr√©sentations</string>
   </property>
   <widget class="QComboBox" name="bCBoxRepresentations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bLabelRepresentations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Spectateur</string>
   </property>
   <widget class="QComboBox" name="bCBoxSpectacteur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bBtnAjouter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Tarif</string>
   </property>
   <widget class="QComboBox" name="bCBoxTarif">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bBtnPaiement">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Paiement</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
